src/cs3resource.py

```diff
@@ -0,0 +1,47 @@
+"""
+cs3resource.py
+
+Resource class to handle CS3 resource references.
+"""
+
+import cs3.storage.provider.v1beta1.resources_pb2 as cs3spr
+
+
+class Resource:
+
+    def __init__(self, end_point: str = "", file_ref: str = "", opaque_id: str = "") -> None:
```

Comment:
Yes, this is quite critical, let's have a chat. A priori, `opaque : bool` can be inferred (currently it is)

Reply:
I proposed to Rasmus to not infer things and make them explicit. We can still infer them in the upper layer, but I would prefer things explicit in the library.

Reply:
The compromise for now is that we infer in the Resource init, and leave explicit the internal class properties.
We also need a method to convert back from the explicit internal properties to the initial string, so that wopi can convert back and forth.

In the future, if we manage to make it explicit in Wopi as well, we just replace the init, remove the string converter and the rest is the same.